src/state/stakeTri/hooks.ts

```diff
@@ -40,4 +42,18 @@ export function useTriBarContract(withSignerIfPossible?: boolean): Contract | nu
     return useContract(chainId && XTRI[chainId].address, TRIBAR_ABI, withSignerIfPossible)
 }
 
-export default useTriBar
+export function useTriBarStats() {
+    const chainId = ChainId.AURORA
+    const totalXTri = useTotalSupply(XTRI[chainId]);
+    const totalTriStaked = useTokenBalance(XTRI[chainId].address, TRI[chainId]);
+
+    const triToXTRIRatio = totalTriStaked != null && totalXTri != null
+        ? totalTriStaked?.divide(totalXTri)
+        : null;
```

Comment:
Worth double-checking this ratio calculation

Reply:
@baboobhaiya to review